lib/knapsack_pro/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module KnapsackPro
-  VERSION = '7.0.0'
+  VERSION = '7.0.1'
```

Comment:
No need to update the gem version inside of the PR.

We increase the gem version after the PR is merged. We create a commit that updates the version on the master branch, and then we create a git tag for that commit.

Sorry, the docs are not clear enough here:
https://github.com/KnapsackPro/knapsack_pro-ruby/blob/master/README.md#publishing
* CHANGELOG.md should be updated in PR
* lib/knapsack_pro/version.rb should be updated after PR is merged, and then you follow the next steps from the readme

Some benefits of this approach:
* while you are testing the gem locally you won't use a version number that has not been released yet. You won't pollute the local API DB or staging/production DB if you hit our API with this unreleased version.
*  when we create a git tag on master, it's for a bumped version commit, so the git history is clear

Reply:
Ah, of course, it makes a lot of sense. I hadn't thought of the implications, e.g. sending a new version to the api when testing.

Reply:
Would you help to update the readme about publishing to make it more clear?

Reply:
Done [here](https://github.com/KnapsackPro/knapsack_pro-ruby/pull/244).